lib/util/glob-util.js

```diff
@@ -8,7 +8,8 @@
 // Requirements
 //------------------------------------------------------------------------------
 
-const fs = require("fs"),
+const _ = require("lodash"),
```

Comment:
We prefer just using `lodash` variable in the rest of the codebase, would you mind switching that up here? Thanks!